compiler-rt/lib/rtsan/tests/rtsan_test_interceptors_posix.cpp

```diff
@@ -1539,6 +1543,7 @@ TEST_F(KqueueTest, KeventDiesWhenRealtime) {
   ExpectNonRealtimeSurvival(Func);
 }
 
+#if SANITIZER_APPLE
```

Comment:
"Kqueue is a scalable event notification interface introduced in FreeBSD 4.1 in July 2000, also supported in NetBSD, OpenBSD, DragonFly BSD, and macOS."

Perhaps we just want `!FREE_BSD` or similar. Also I would potentially un-nest these `ifdefs` for clarity

Reply:
note that s the kevent64 flavour only.

Reply:
ah, fine to leave then probably. Thanks for pointing that out